code/LINDA/LINDA_turf_tile.dm

```diff
@@ -66,7 +66,7 @@ turf/simulated/New()
 
 turf/simulated/Del()
 	if(active_hotspot)
-		qdel(active_hotspot)
+		active_hotspot.Kill()
```

Comment:
Um. With this and making Kill do nothing but place the hotspot in the pool, you could end up leaving references to the hotspot, qdel here was right.